cli/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "immich-cli",
+  "dependencies": {
+    "axios": "^1.4.0",
+    "form-data": "^4.0.0",
+    "mime-types": "^2.1.35",
+    "systeminformation": "^5.18.4"
+  },
+  "devDependencies": {
+    "@types/chai": "^4.3.5",
+    "@types/chokidar": "^2.1.3",
+    "@types/jest": "^29.5.2",
+    "@types/js-yaml": "^4.0.5",
+    "@types/mime-types": "^2.1.1",
+    "@types/mock-fs": "^4.13.1",
+    "@types/node": "^20.3.1",
+    "@typescript-eslint/eslint-plugin": "^5.60.1",
+    "chai": "^4.3.7",
+    "commander": "^11.0.0",
+    "eslint": "^8.43.0",
+    "eslint-config-prettier": "^8.8.0",
+    "eslint-plugin-jest": "^27.2.2",
+    "eslint-plugin-prettier": "^4.2.1",
+    "eslint-plugin-unicorn": "^47.0.0",
+    "glob": "^10.3.1",
+    "immich-sdk": "^1.62.0",
```

Comment:
Now that the CLI is in the same repo, can we make the openAPI generator generates the SDK directly in the repo, similar to the web? This would keep the API in sync with the server changes.

Reply:
Yes, we should do that, but I'm not sure where to start. Do you want to give me a pointer or two?

Reply:
Will do!

Reply:
Done